contracts/extensions/VotingReputation.sol

```diff
@@ -442,7 +447,24 @@ contract VotingReputation is DSMath, PatriciaTreeProofs {
     poll.skillId = newDomainSkillId;
     poll.skillRep = getReputationFromProof(_pollId, address(0x0), _key, _value, _branchMask, _siblings);
 
+    if (poll.votes[NAY] < poll.votes[YAY]) {
+      poll.stakes[NAY] = sub(poll.stakes[NAY], poll.paidVoterComp);
+    } else {
+      poll.stakes[YAY] = sub(poll.stakes[YAY], poll.paidVoterComp);
+    }
+
+    delete poll.paidVoterComp;
+
     emit PollEscalated(_pollId, msg.sender, _newDomainId);
+
+    // Check to see if the stake is unchanged, if so skip the staking period
+    if (poll.stakes[NAY] == getRequiredStake(_pollId)) {
```

Comment:
Surely you have to check both sides here? I think this is now only possible if there is no voter compensation, though, if you're subtracting the voter compensation from the losing side now.

Reply:
Ah yes, I hadn't considered the interaction between this and the stake subtraction for voter comp.

Reply:
Although wait -- if we've had a vote, we have ostensibly removed someone's stake, meaning we need a staking period anyway, even if the amount of reputation is the same, simply to top-up whoever paid out the voter comp. So this isn't needed!

Reply:
As I say, this is still relevant in the case where there is no voter compensation (i.e. `_voterRewardFraction` = 0).

Reply:
Yes you are correct.